.circleci/config.yml

```diff
@@ -43,6 +43,25 @@ jobs:
           root: .
           paths:
             - ./*
+  integration-test:
+    working_directory: ~/addons-frontend
+    machine: true
+    steps:
+      - checkout
+      - run:
+          name: Install Docker Compose
+          command: |
+            set -x
+            pip install docker-compose>=1.18
+            docker-compose --version
+      - run:
+          name: Install tox
+          command: pip install tox
+      - run:
+          name: Run tests
+          command: tox -e ui-test-ci
```

Comment:
Why do we need tox for this? Can't you just move the commands from `tox.ini` in here? What does value does `tox` add?

Reply:
Tox runs other commands, like the discopane tests.